clients/ui/frontend/src/app/pages/modelCatalog/screens/ModelCatalogPage.tsx

```diff
@@ -39,61 +34,79 @@ const ModelCatalogPage: React.FC = () => {
     );
   }
 
-  if (error) {
+  if (catalogModelsLoadError) {
     return (
       <PageSection>
         <Alert variant="danger" title="Failed to load model catalog" isInline>
-          {error.message}
-          <Button variant="link" onClick={refreshSources}>
+          {catalogModelsLoadError.message}
+          <Button variant="link" onClick={refresh}>
             Try again
           </Button>
         </Alert>
       </PageSection>
     );
   }
 
-  if (sources.length === 0) {
-    return (
-      <PageSection>
-        <EmptyState>
-          <CubesIcon />
-          <Title headingLevel="h4" size="lg">
-            No models available
-          </Title>
-          <EmptyStateBody>
-            There are no models available in the catalog. Try refreshing or contact your
-            administrator.
-          </EmptyStateBody>
-          <Button variant="primary" onClick={refreshSources}>
-            Refresh
-          </Button>
-        </EmptyState>
-      </PageSection>
-    );
-  }
-
   return (
-    <PageSection>
-      <Title headingLevel="h1" size="2xl" className="pf-v5-u-mb-md">
-        Model Catalog
-      </Title>
-      <p className="pf-v5-u-mb-lg">
-        Discover models that are available for your organization to register, deploy, and customize.
-      </p>
-      <Gallery hasGutter minWidths={{ default: '300px' }}>
-        {sources.map((source) =>
-          (source.models || []).map((model) => (
-            <GalleryItem key={model.id}>
+    <>
+      <ScrollViewOnMount shouldScroll />
+      <ApplicationsPage
+        title={
+          <TitleWithIcon
+            title="Model Catalog"
+            // for now, added the modelRegistrySettings for this - will remove once we update the shared library
+            objectType={ProjectObjectType.modelRegistrySettings}
```

Comment:
Will raise PR to update the shared library to include Model catalog img, then update this.